src/database/hooks/useSaveAnonChatHook.ts

```diff
@@ -34,13 +34,14 @@ const useSaveAnonChatHook = () => {
     object: InitAnonymousChatData,
     status = 'sent',
     withNavigate,
-    type: string
+    type?: string
   ) => {
     if (!localDb) return;
 
     const chatName = await getAnonymousChatName(object?.members);
     const initAnonymousChat: ModifyAnonymousChatData = {
       ...object,
+      reply_data: object.message?.reply_data,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `type` parameter in the function signature has been made optional. This could potentially introduce bugs if the function is called without this argument and the function implementation expects it to be present. Please ensure that the function handles the case where `type` is undefined.
```diff
-    type?: string
+    type: string
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->